src/apps/btc/btc_common.h

```diff
@@ -167,4 +160,22 @@ USE_RESULT bool btc_common_pkscript_from_multisig(
     uint8_t* script_out,
     size_t* script_out_size);
 
+/**
+ * Validate a m-of-n multisig account. This includes checking that:
+ * - 0 < m <= n <= 15
+ * - the keypath conforms to bip48 for p2wsh: m/48'/coin'/account'/2'
+ * - one of the xpubs is ours (the currenty unlocked keystore).
+ * @param[in] multisig Multisig configuration (threshold, signers). The xpubs are account-level
+ * xpubs.
+ * @param[in] keypath address-level keypath, e.g. m/48'/0'/10'/2'/0/10
```

Comment:
I'm a bit confused here. The doc says here that a full keypath is expected, but then later you check this same keypath in `_is_valid_keypath_account_multisig_p2wsh` and restrict it to a size of four, seemingly without changing its size within `btc_common_multisig_is_valid`. Am I missing something here?

Reply:
This line in the docs is wrong :O the description a few lines further up is correct. i'll amend.

Reply:
Done now.